doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -784,3 +785,345 @@ JCL      - d247952 based on jdk-11.0.6+6)
+
+[1] Run the following command to get the source of the cross-toolchain:
+```
+$ git clone --recursive https://github.com/riscv/riscv-gnu-toolchain.git
+```
+
+[2] When the process is complete, install all required software packages on your host system:
+```
+$ sudo apt-get install \
+    autoconf \
+    automake \
+    autotools-dev \
+    curl \
+    libmpc-dev \
+    libmpfr-dev \
+    libgmp-dev \
+    gawk \
+    build-essential \
+    bison \
+    flex \
+    texinfo \
+    gperf \
+    libtool \
+    patchutils \
+    bc \
+    zlib1g-dev \
+    libexpat-dev
+```
+
+[3] Set up the installation path and build the cross-toolchain on your system:
+```
+cd riscv-gnu-toolchain
+./configure --prefix=/opt/riscv_toochain_linux  #the path to install the cross-toolchain
+make linux
+```
+
+:bulb:
+Please refer to the latest instructions at https://github.com/riscv/riscv-gnu-toolchain for details if
+you are working on other host systems or if any issue is detected when compiling the cross-toolchain.
+
+[4] Run the RISC-V specific gcc command to ensure the cross-toolchain works good as expected:
+```
+$ riscv64-unknown-linux-gnu-gcc --version
+riscv64-unknown-linux-gnu-gcc (GCC) 9.2.0
+Copyright (C) 2019 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.
+There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+```
+
+### 4. Get the source for building the JDK
+:rocket: You need to clone the Extensions for OpenJDK for OpenJ9 project, which is a git mirror of OpenJDK without the HotSpot JVM, but with an **openj9** branch that contains a few necessary patches. Run the following command:
+```
+git clone https://github.com/ibmruntimes/openj9-openjdk-jdk11.git
+```
+Cloning this repository can take a while because OpenJDK is a large project! When the process is complete, change directory into the cloned repository:
+```
+cd openj9-openjdk-jdk11
+```
+Now fetch additional sources from the Eclipse OpenJ9 project and its clone of Eclipse OMR:
+```
+bash get_source.sh
+```
+
+### 5. Configure the cross-compiling environment
+:rocket:
+When you have all the source files on the host system, run the following configure command to set up the cross-compilation environment:
+```
+export JAVA_HOME=/<path_to_boot_JDK>
+export RISCV64=/<path_to_gnu_cross_toolchain>
+export PATH="$RISCV64/bin:$JAVA_HOME/bin:$PATH"
+
+bash configure --disable-warnings-as-errors \ 
+               --disable-ddr \
+               --with-boot-jdk=/<path_to_boot_JDK> \
+               --with-build-jdk=/<path_to_boot_JDK> \
+               --openjdk-target=riscv64-unknown-linux-gnu \
+               --with-sysroot=/<path_to_your_fedora_mount_directory> \
+               --with-freemarker-jar=/<path_to>/freemarker.jar
+```
+
+:pencil:
+If you want to build an OpenJDK that uses OpenSSL, you must specify `--with-openssl={path_to_library}`
```

Comment:
Please use `<>` instead of `{}` to mark placeholders for consistency.

Reply:
Fixed as suggested.